StarCellBio/views.py

```diff
@@ -84,13 +84,20 @@ def get_model(request):
     return response
 
 
+def get_account_type(request):
```

Comment:
This appears to be a duplicate function using slightly different logic, it is probably worth consolidating